scio-bigtable/src/it/scala/com/spotify/scio/bigtable/BigtableIT.scala

```diff
@@ -0,0 +1,92 @@
+/*
+ * Copyright 2017 Spotify AB.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package com.spotify.scio.bigtable
+
+import com.google.bigtable.v2.{Mutation, Row}
+import com.google.cloud.bigtable.config.BigtableOptions
+import com.google.cloud.bigtable.grpc.BigtableClusterUtilities
+import com.google.protobuf.ByteString
+import com.spotify.scio.ScioContext
+import com.spotify.scio.testing.PipelineSpec
+import org.joda.time.Duration
+
+object BigtableIT {
+  val projectId = "data-integration-test"
+  val instanceId = "scio-bigtable-it"
+  val clusterId = "scio-bigtable-it-cluster"
+  val zoneId = "us-east1-b"
+  val tableId = "scio-bigtable-it-counts"
+  val testData = Seq(("key1", 1L), ("key2", 2L), ("key3", 3L))
```

Comment:
Prefix keys with a single UUID or something? In case there're multiple IT running.